### PR TITLE
Support for messages and tokens performance tests

### PR DESCRIPTION
**Pull request by eberger727**
Full support for broadcast/private message/mint/burn/transfer for performance testing

```shell
Commands:
  get_txs, msg_broadcast, msg_private, token_burn, token_mint, token_transfer

Flags:
  -f, --frequency int          Requests Per Second (RPS) of each worker (default 10)
  -h, --help                   help for ff-perf
  -d, --jobDuration duration   Duration of each job done by worker (default 1m0s)
  -l, --length duration        Length of entire performance test (default 1m0s)
      --longMessage            Include long string in message
  -n, --node string            FireFly node endpoint to test (default "http://localhost:5000")
  -r, --recipient string       Recipient for FireFly messages
      --tokenMessage           Attach message to token
      --tokenType string       [fungible nonfungible] (default "fungible")
  -w, --workers int            Number of workers at a time (default 1)
```

### CODE REVIEW

File: internal/perf/get_txs.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package perf
+
+import "github.com/hyperledger/firefly-perf-cli/internal/conf"
+
+func (pr *perfRunner) RunGetTransactions(id int) {
+	targeter := pr.getApiTargeter("GET", "transactions", "")
+	pr.runAttacker(targeter, id, conf.PerfCmdGetTransactions.String())
```
Comment (awrichar) [inline internal/perf/get_txs.go:7]: Attacker seems aggressive 🙃 this is a perf test after all. Fine line between performance and stress though I suppose.

**Comment by awrichar** [comment]
Did you mean to check in a binary "main" file?

**Review by awrichar — APPROVED** [review]
Delete the `main` binary unless it's needed for some reason... but otherwise looks good.

File: internal/perf/token_mint.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	targeter := pr.getApiTargeter("POST", "tokens/mint", payload)
+	pr.runAttacker(targeter, id, conf.PerfCmdTokenMint.String())
+}
+
+func (pr *perfRunner) MintTokensForTransfer(transferType string) error {
+	log.Infof("Minting tokens to %s", transferType)
+
+	body := map[string]interface{}{
+		"pool":   pr.poolName,
+		"amount": "1000000000000",
```
Comment (awrichar) [inline internal/perf/token_mint.go:38]: This is a lot

Comment (eberger727) [reply internal/perf/token_mint.go]: removed

File: internal/perf/get_txs.go (hunk 1 above)
Comment (eberger727) [reply internal/perf/get_txs.go]: Removed to only submit one request

**Pull request merged**